pkg/gardenlet/controller/shoot/shoot_care_control.go

```diff
@@ -134,7 +134,7 @@ func (c *defaultCareControl) Care(shootObj *gardencorev1beta1.Shoot, key string)
 
 	shootLogger.Debugf("[SHOOT CARE] %s", key)
 
-	operation, err := operation.New(shoot, c.config, shootLogger, c.k8sGardenClient, c.k8sGardenCoreInformers, c.identity, c.secrets, c.imageVector)
+	operation, err := operation.New(shoot, shoot.Spec.SeedName, c.config, shootLogger, c.k8sGardenClient, c.k8sGardenCoreInformers, c.identity, c.secrets, c.imageVector)
```

Comment:
Why is the seedname a separate argument? You already pass the `shoot` in the first argument - isn't this enough?

Reply:
Well, idk if it is the right way to achieve this, but in order to pass the seed Name of the source Seed you have to extract it form the Shoot.Status.SeedName. Previously the seedName was hardcoded in the operation by taking it from the Spec.SeedName.